sway-ir/src/variable.rs

```diff
@@ -67,3 +67,64 @@ impl LocalVar {
         context.local_vars[self.0].mutable = mutable;
     }
 }
+
+/// A wrapper around an [ECS](https://github.com/orlp/slotmap) handle into the
+/// [`Context`].
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, DebugWithContext)]
+pub struct GlobalVar(#[in_context(global_vars)] pub slotmap::DefaultKey);
+
+#[doc(hidden)]
+#[derive(Clone, DebugWithContext)]
+pub struct GlobalVarContent {
+    pub ptr_ty: Type,
+    pub initializer: Option<Constant>,
+    pub mutable: bool,
+}
+
+impl GlobalVar {
+    /// Return a new Global of a specific type with an optional [`Constant`] initializer.  If a
```

Comment:
Just for the consistency with other comments, to use either local or [GlobalVar].
```suggestion
    /// Return a new global of a specific type with an optional [`Constant`] initializer.  If a
```